holo-utils/src/bier.rs

```diff
@@ -13,11 +13,98 @@ use derive_new::new;
 use holo_yang::{ToYang, TryFromYang};
 use ipnetwork::IpNetwork;
 use serde::{Deserialize, Serialize};
+use tracing::warn;
 
 use crate::ip::AddressFamily;
 
 pub type SubDomainId = u8;
 pub type BfrId = u16;
+pub type SetIdentifier = u8;
+pub type Bift = HashMap<
+    (SubDomainId, IpAddr, SetIdentifier),
+    (Bitstring, Vec<(BfrId, IpAddr)>, u32, String),
+>;
+
+#[cfg(feature = "fastclick")]
+pub async fn bift_sync_fastclick(bift: &Bift) {
+    for ((_sd_id, nbr, _si), (bs, ids, idx, name)) in bift.iter() {
+        /* List the position of bits that are enabled in the bitstring, this is required by the
```

Comment:
nit: let's stick to C++ style comments (`//`) for consistency with the rest of the codebase. Also, let's try to keep comments within 80 characters for readability.

*rustfmt* has options to handle that automatically (`normalize_comments` and `wrap_comments`), but unfortunately we can't enable them since they are still a bit unstable and unreliable.